protocol/examples/messageData/coin/post_transaction_bad_signature.json

```diff
@@ -1,7 +1,7 @@
 {
     "object": "coin",
     "action": "post_transaction",
-    "transaction_id": "ru0acQHE0QEqkA1IgdjTlqIwYxyq3YgN1M6pj9A2FF8=",
+    "transaction_id": "R5bU6MLWtJasGCJSnV73GBsinYajGsuai0sTNau8nv8=",
```

Comment:
Could this influence the tests in Go? If so they shouldn't be changed there as well (if not already changed  in the other PR of course)

Reply:
let me double check

Reply:
It just does `require.Error(t, err)` so it's actually unspecific just like the old Scala one